src/controllers/ArticleLikeController.js

```diff
@@ -0,0 +1,63 @@
+import status from '../config/status';
```

Comment:
the like and dislike functionality can be placed in one file, is there a reason you feel they should be separated?

Reply:
Yeap.. I wanted to separate them, so that reviewers will not be confused, but I can change them..